### PR TITLE
Jacob/html

### PR DESCRIPTION
**Pull request by ShaveYourEyebrows**
Add my HTML in pls

### CODE REVIEW

File: OBlock/html/frontend.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang = 'en'>
+	<head>
+		<meta charset='UTF-8'>
+		<meta name='viewport' content='width=device-width, initial-scale=1.0'>
+		<link rel="icon" href = "shrek.png" type= "image/png"/>
```
Comment (oOoOoOo0o) [inline OBlock/html/frontend.html]: Let's not use Shrek

File: OBlock/html/shrek.png
Comment (oOoOoOo0o) [inline OBlock/html/shrek.png:1]: No Shrek

**Review by oOoOoOo0o — CHANGES_REQUESTED** [review]

File: OBlock/html/frontend.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang = 'en'>
```
Comment (Bourkupine) [inline OBlock/html/frontend.html]: lets remove the spacing around the '=' symbol for consistency

File: OBlock/html/frontend.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+		<link rel="icon" href = "shrek.png" type= "image/png"/>
+		<title>A1 Please</title>    
+	</head>
+	<body>
+		<p style = "color:cyan;font-size:30px;outline:3px;-webkit-text-stroke-width: 3px;">Welcome to the O-Block Project Website</p>
+		<form name="test">
+		<input type = "text" name = "studentname" placeholder="Enter the student name pls">
+		<input type = "int" name = "studentid" placeholder="Enter the student ID">
+		<input type = "int" name = "courseid" placeholder="Enter the course ID they want to enroll in">
+		<br></br>
```
Comment (Bourkupine) [inline OBlock/html/frontend.html]: dont think </br> is needed

File: OBlock/html/shrek.png
Comment (Bourkupine) [reply OBlock/html/shrek.png:1]: yeah lets not use shrek

File: OBlock/html/init.txt
Comment (Bourkupine) [inline OBlock/html/init.txt:1]: remove this now

**Review by Bourkupine — CHANGES_REQUESTED** [review]
Looks good, just minor nit picks

File: OBlock/html/frontend.html (hunk 3 above)
Comment (Bourkupine) [reply OBlock/html/frontend.html]: forgot comments were md, meant the /br tag

**Review by carthy27 — CHANGES_REQUESTED** [review]
no shrek grrrr

**Review by oOoOoOo0o — APPROVED** [review]

**Review by MartyD1 — APPROVED** [review]

**Pull request closed without merging**